tests/conftest.py

```diff
@@ -2,47 +2,127 @@
     Conftest file for pytest fixtures
 """
 import datetime as dt
+import logging
+import shutil
 from pathlib import Path
 
 import pytest
 import ruamel.yaml
 from trailblazer.mip import files as mip_dna_files_api
 
-from cg.apps.madeline.api import MadelineAPI
+# from cg.apps.hk import HousekeeperAPI
 from cg.apps.mip_rna import files as mip_rna_files_api
 from cg.meta.store import mip_rna as store_mip_rna
 from cg.store import Store
 
-pytest_plugins = [
-    "tests.apps.lims.conftest",
-    "tests.apps.loqus.conftest",
-    "tests.apps.crunchy.conftest",
-    "tests.cli.conftest",
-    "tests.delivery.conftest",
-    "tests.delivery.conftest",
-    "tests.meta.compress.conftest",
-    "tests.meta.conftest",
-    "tests.meta.orders.conftest",
-    "tests.meta.report.conftest",
-    "tests.meta.transfer.conftest",
-    "tests.meta.upload.conftest",
-    "tests.store.conftest",
-    "tests.apps.mutacc_auto.conftest",
-]
+from .mocks.hk_mock import MockHousekeeperAPI
+from .mocks.madeline import MockMadelineAPI
+from .store_helpers import Helpers
 
 CHANJO_CONFIG = {"chanjo": {"config_path": "chanjo_config", "binary_path": "chanjo"}}
 CRUNCHY_CONFIG = {
     "crunchy": {
         "cram_reference": "/path/to/fasta",
-        "slurm": {"account": "mock_account", "mail_user": "mock_mail", "conda_env": "mock_env",},
+        "slurm": {
+            "account": "mock_account",
+            "mail_user": "mock_mail",
+            "conda_env": "mock_env",
+        },
     }
 }
 
+LOG = logging.getLogger(__name__)
+
+
+# Case fixtures
+
+
+@pytest.fixture(name="case_id")
+def fixture_case_id():
+    """Return a case id"""
+    return "yellowhog"
+
+
+@pytest.yield_fixture(scope="function", name="family_name")
+def fixture_family_name() -> str:
+    """Return a family name"""
+    return "family"
+
+
+@pytest.fixture(scope="function", name="analysis_family_single_case")
+def fixture_analysis_family_single(case_id, family_name):
+    """Build an example family."""
+    family = {
+        "name": family_name,
+        "internal_id": case_id,
+        "data_analysis": "mip",
+        "application_type": "wgs",
+        "panels": ["IEM", "EP"],
+        "samples": [
+            {
+                "name": "proband",
+                "sex": "male",
+                "internal_id": "ADM1",
+                "status": "affected",
+                "ticket_number": 123456,
```

Comment:
Yes I agree, this problem is more complicated. There was a PR merged from IS while I was working that broke my code. I had to adress this by adding a fix. It can be a bit complicated to get your suggestion to propagate all the way through